mon-pix/app/controllers/terms-of-service-pole-emploi.js

```diff
@@ -22,14 +29,19 @@ export default class TermsOfServicePeController extends Controller {
   @action
   async submit() {
     if (this.isTermsOfServiceValidated) {
-      this.showErrorTermsOfServiceNotSelected = false;
+      this.errorMessage = false;
```

Comment:
```suggestion
      this.errorMessage = null;
```
Pour ne pas mélanger string et boolean, et être cohérent avec la valeur par défaut.

Reply:
Oui c'est juste, merci c'est corrigé !